2022-05-20 13:36:10,896 INFO  ==============================================================
2022-05-20 13:36:10,897 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
2022-05-20 13:36:10,898 INFO  Dir:   tedge_mapper_c8y/
2022-05-20 13:36:10,899 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
2022-05-20 13:36:10,900 INFO  ==============================================================
2022-05-20 13:36:11,468 INFO  Executed sudo<tedge_mapper_c8y>, exit status 0
2022-05-20 13:36:11,480 INFO  Started sudo<tedge_sub> with process id 26473
2022-05-20 13:36:12,146 INFO  Executed sudo<tedge_msg>, exit status 0
2022-05-20 13:36:12,711 INFO  Executed sudo<tedge_pub>, exit status 0
2022-05-20 13:36:13,224 INFO  Executed sudo<kill_out>, exit status 0
2022-05-20 13:36:13,264 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
2022-05-20 13:36:13,270 INFO  
2022-05-20 13:36:13,271 INFO  cleanup:
2022-05-20 13:36:13,272 INFO  mapper_cleanup
2022-05-20 13:36:13,884 INFO  Executed sudo<tedge_mapper_c8y>, exit status 0
2022-05-20 13:36:13,888 INFO  
2022-05-20 13:36:13,889 INFO  Test duration: 3.00 secs
2022-05-20 13:36:13,890 INFO  Test final outcome:  FAILED
2022-05-20 13:36:13,891 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
2022-05-20 13:36:13,892 INFO  
